package.json

```diff
@@ -18,7 +18,7 @@
     "serve-pkgs": "./shell/scripts/serve-pkgs",
     "publish-shell-reset-reg": "cd shell && npm publish",
     "clean": "./shell/scripts/clean",
-    "lint": "./node_modules/.bin/eslint --max-warnings 0 --ext .js,.ts,.vue .",
+    "lint": "./node_modules/.bin/eslint --rulesdir ./eslint-local-rules --max-warnings 0 --ext .js,.ts,.vue .",
```

Comment:
Do you know if we can specify `rulesdir` in `eslintrc` instead of with a cli flag? 

My assumption is that it's best to stick with the approach in this PR, but my curiousity was piqued when I found some old conversations that suggests eslint deprecating the usage of `ruledir` and recommending `eslint-plugin-rulesdir` as the preferred approach[^1][^2]. I'm guessing that this never materialized in any meaningful way, seeing how we are using `rulesdir` today.

[^1]: https://github.com/eslint/eslint/issues/2715
[^2]: https://github.com/eslint/eslint/issues/8769

Reply:
I'm not sure. I'll give it a try.

Reply:
We cannot specify `rulesdir` in `eslintrc`.

It failed with 
![image](https://github.com/user-attachments/assets/1fe2dfff-402f-4ffe-bd41-3a34aba4d635)

Which lead me to go look at the config-validator code and there isn't a `rulesdir` option.
https://github.com/eslint/eslintrc/blob/v0.4.3/conf/config-schema.js

When we upgrade eslint we will likely want to switch from `--rulesdir` to `--plugin` which will cause us to create one extra file to create the plugin with all of our rules. https://eslint.org/docs/latest/use/command-line-interface#--plugin